web_common/web_common/templates/header.html

```diff
@@ -3,24 +3,19 @@
     <b>/</b>
   </a>
 
-  {% if userdata['developer'] == 1 %}
+  {% if userdata['is_developer'] == 1 %}
     <div class="header-dropdown">
       <div class="header-dropdown-item-margin">
-	<a href="{{ notebook_base_url }}/notebook" class="header-dropdown-link">Notebook</a>
+	<span class="disabled-header-dropdown-item">Auth</span>
```

Comment:
What’s the purpose of this?

Reply:
Purpose of what?  It is a drop down link to the users page.

Reply:
But it isn't a link and the class is disabled?

Reply:
I see, this is the placeholder for hover state. I don't like this because it's inconsistent with other dropdown (right below this, the top level item is Batch and is a link).

Reply:
Yes, that's right.  Auth has no page of its own, so it is disabled (like monitoring).

Reply:
It is exactly like the monitoring dropdown.  If the service has a suitable root page, the header item is active and links to the main page.  If it does not, it is disabled and just used for the dropdown.

I could just make Auth point to the user admin page but that did not seem appropriate.

Reply:
I'm slightly biased against dropdowns for this reason. I think we should have consistent UI, so either always have the top level item clickable in a dropdown, or not. Dropdown can be indicated with an arrow to distinguish from links with no children.